### PR TITLE
Merge postcss-modules config

### PR DESCRIPTION
**Pull request by isbasex**

### CODE REVIEW

File: src/transforms/postcss.js [hunk 1]
```diff
@@ -24,12 +26,16 @@ async function getConfig(asset) {
   }
 
   config = config || {};
-  config.plugins = loadPlugins(config.plugins, asset.name);
+  const postcssModulesConfig = Object.assign(get(config.plugins, 'postcss-modules') || {}, {
```
Comment (thejameskyle) [inline src/transforms/postcss.js]: Why not just `config.plugins['postcss-modules']`?

Comment (isbasex) [reply src/transforms/postcss.js]: type safety

File: package.json [hunk 2]
```diff
@@ -23,6 +23,8 @@
     "htmlnano": "^0.1.6",
     "is-url": "^1.2.2",
     "js-yaml": "^3.10.0",
+    "lodash.get": "^4.4.2",
+    "lodash.omit": "^4.5.0",
```
Comment (devongovett) [inline package.json]: I'd rather avoid adding more dependencies if we can. omit could just be `delete config.plugins['postcss-modules']`, and get could be just be `config.plugins['postcss-modules']`.

**Comment by isbasex** [comment]
@devongovett Now, just pure js

**Pull request merged**